plugin/evm/message/codec.go

```diff
@@ -21,8 +21,8 @@ var (
 )
 
 func init() {
-	Codec = codec.NewManager(maxMessageSize)
-	c := linearcodec.NewDefault()
+	Codec = codec.NewManager(maxMessageSize + wrappers.ShortLen) // Add room for the codec version
```

Comment:
I'd prefer to revert this `+ wrappers.ShortLen`. It isn't needed and including it makes it seem like it is.